consensus/hotstuff/verification/combined_signer_v2_test.go

```diff
@@ -150,21 +172,35 @@ func TestCombinedSignWithNoBeaconKey(t *testing.T) {
 	// for this failed node, which can be used to verify signature from
 	// this failed node.
 	committee.On("DKG", mock.Anything).Return(dkg, nil)
+	committee.On("Self").Return(me.NodeID())
+	committee.On("IdentityByBlock", fblock.Header.ID(), mock.Anything).Return(ourIdentity, nil)
```

Comment:
Since we are always returning our identity, shouldn't the expected input be `signerID`?

```suggestion
	committee.On("IdentityByBlock", fblock.Header.ID(), signerID).Return(ourIdentity, nil)
```